### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sidravi1**
Reviewer: @Tanmay-97 

Estimate: 10 mins

----

# Update README.md

## Ticket

Fixes: [AAQ-156](https://idinsight.atlassian.net/browse/AAQ-156)

## Description, Motivation and Context

Add more detail to the landing page of the repo.

### Changes
* Updated README.md
* Updated docstring for `llm-response`

## How Has This Been Tested?

See [README.md](https://github.com/IDinsight/aaq-core/tree/readme-contributing) rendered on the branch

## Checklist:

This checklist is a useful reminder of small things that can easily be forgotten.
Put an `x` in all the items that apply and remove any items that are not relevant to this PR.

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [x] I have updated the automated tests (if applicable)
- [x] I have written [good commit messages][1]
- [x] I have updated the README file (if applicable)
- [x] I have updated affected documentation (if applicable)

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,135 @@
...
+ (___/    \___)(___/    \___)\"____/\__\
+ASK-A-QUESTION
+</pre>
+
+<p align="center" style="text-align:center">
+<a href="https://idinsight.github.io/aaq-core/">Docs</a> |
+<a href="#features">Features</a> |
+<a href="#usage">Usage</a> |
+<a href="#architecture">Architecture</a> |
+<a href="#partners">Partners</a>
```
Comment (Tanmay-97) [inline README.md:17]: This section is TBD?
Should we make it "Funders and Partners" and just add Google.org for now; till Reach feels okay to be included in this list?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,135 @@
...
+- [ ] Hindi
+- [ ] Igbo
+
+#### **:speech_balloon: conversation capability**
+
+Refine or clarify your question through conversation
+
+#### :video_camera: multimedia content
+
+Respond with not just text but images and videos as well.
```
Comment (Tanmay-97) [inline README.md:57]: This isn't on the roadmap; should this be about voicenotes?
Re: images, content could be having URLs for images which we can render on WhatsApp easily

File: README.md [hunk 3]
```diff
@@ -1,2 +1,135 @@
...
+
+#### :rotating_light: urgency detection
```
Comment (Tanmay-97) [inline README.md:59]: Maybe "Text Classification" or "Intent Tagging" could be a feature, and UD could be a use-case of that feature - depends on how we've structured the code

File: README.md [hunk 4]
```diff
@@ -1,2 +1,135 @@
...
+    /    \        /    \    // ____  \
+    /' /\  \      /' /\  \  /  /    )  )
+  //  __'  \    //  __'  \(: (____/ //
+ /   /  \\  \  /   /  \\  \\         \
+ (___/    \___)(___/    \___)\"____/\__\
+ASK-A-QUESTION
+</pre>
+
+<p align="center" style="text-align:center">
+<a href="https://idinsight.github.io/aaq-core/">Docs</a> |
```
Comment (Tanmay-97) [inline README.md:13]: When we say `Docs` do we mean `API Reference`? Should we name this better then? Like `Developer Documentation` or `Quick Start`

File: README.md [hunk 5]
```diff
@@ -1,2 +1,135 @@
...
+<a href="#features">Features</a> |
+<a href="#usage">Usage</a> |
+<a href="#architecture">Architecture</a> |
+<a href="#partners">Partners</a>
+</p>
+
+**[Ask-a-question](https://idinsight.github.io/aaq-core/) allows your beneficiaries or staff to ask question in natural language in their preferred language and receive answers from your content repository**
```
Comment (Tanmay-97) [inline README.md:20]: My super concise suggestion - **Ask A Question lets your users ask questions naturally and receive answers from your content repository**

File: README.md [hunk 6]
```diff
@@ -1,2 +1,135 @@
...
+#### **:earth_africa: Support for low resourced language**
+
+Ask questions in local languages. Languages currently on the roadmap
+
+- [ ] Xhosa
+- [ ] Zulu
+- [ ] Hindi
+- [ ] Igbo
+
+#### **:speech_balloon: conversation capability**
```
Comment (Tanmay-97) [inline README.md:51]: Would suggestion Title case for these headers

**Review by Tanmay-97 — APPROVED** [review]
Looks good, some minor improvements suggested

File: README.md (hunk 1 above)
Comment (sidravi1) [reply README.md:17]: Good idea!

**Pull request merged**